timesketch/api/v1/__init__.py

```diff
@@ -11,7 +11,4 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from django.test import TestCase
-
-# Create your tests here.
+"""This is version 1 of the Timesketch API."""
```

Comment:
The user of the API is the frontend client, e.g. the browser. Resources and API URL mappings are defined when initializing the application, i.e. /api/v1/.. The client always uses a specific version of the API. We can always add a get_version() in the resource MixIn. I'll do that in a later PR is needed.